src/DependencyInjection/KnpDisqusExtension.php

```diff
@@ -42,9 +43,5 @@ public function load(array $configs, ContainerBuilder $container)
         if (isset($config['base_url'])) {
             $container->setParameter('knp_disqus.base_url', $config['base_url']);
         }
```

Comment:
Instead of setting parameters for api_key, secret_key and base_url, let's fetch the `Disqus` service definition directly and modify its arguments.